.github/ISSUE_TEMPLATE/translation.yml

```diff
@@ -31,10 +31,9 @@ body:
     attributes:
       value: |
         If this is a new translation, please complete the following items:
-        - [ ] Enable i18n by update website/config.toml and website/i18n/<short-language-code>.toml
-        - [ ] Copy all contents to content/<short-language-code>
+        - [ ] Enable i18n by update website/config.toml and website/i18n/<short-language-code>.toml>
 
         All translations require the following items:
-        - [ ] Translate some key pages (e.g. content/<short-language-code>/_index.md)
+        - [ ] Translate some key pages (e.g. content/_short-language-code_/_index.md)
```

Comment:
Out of curiosity, why the move to italics here? I always thought `<>` was to indicate something the user had to type and may be an easier convention for most people?

Reply:
with the <> it's not being rendered correctly. Here's what it looks like atm:
<img width="417" alt="Screenshot 2024-04-13 at 12 47 37" src="https://github.com/cncf/tag-app-delivery/assets/2743542/0dcdc6dd-d2db-44f2-af93-d73f08684929">

Reply:
Doh. Great shout! I suspect it is about escaping them but definitely not worth more energy for something so simple 👍